hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCheckAccess.java

```diff
@@ -88,11 +90,12 @@ private FileSystem getTestUserFs() throws Exception {
     getRawConfiguration()
         .setBoolean(AZURE_CREATE_REMOTE_FILESYSTEM_DURING_INITIALIZATION,
             orgCreateFileSystemDurungInit);
-    return fs;
+    this.testUserFs = fs;
   }
 
-  @Test(expected = IllegalArgumentException.class)
```

Comment:
It is expected to throw IllegalArgumentException.

Reply:
Done